zerver/views/user_settings.py

```diff
@@ -171,30 +169,20 @@ def update_display_settings_backend(
-                                enable_stream_push_notifications=REQ(validator=check_bool,
-                                                                     default=None),
-                                enable_stream_sounds=REQ(validator=check_bool,
-                                                         default=None),
-                                enable_desktop_notifications=REQ(validator=check_bool,
-                                                                 default=None),
-                                enable_sounds=REQ(validator=check_bool,
-                                                  default=None),
-                                enable_offline_email_notifications=REQ(validator=check_bool,
-                                                                       default=None),
-                                enable_offline_push_notifications=REQ(validator=check_bool,
-                                                                      default=None),
-                                enable_online_push_notifications=REQ(validator=check_bool,
-                                                                     default=None),
-                                enable_digest_emails=REQ(validator=check_bool,
-                                                         default=None),
-                                pm_content_in_desktop_notifications=REQ(validator=check_bool,
-                                                                        default=None)):
-    # type: (HttpRequest, UserProfile, Optional[bool], Optional[bool], Optional[bool], Optional[bool], Optional[bool], Optional[bool], Optional[bool], Optional[bool], Optional[bool], Optional[bool], Optional[bool]) -> HttpResponse
+def json_change_notify_settings(
+        request: HttpRequest, user_profile: UserProfile,
+        enable_stream_desktop_notifications: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_stream_email_notifications: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_stream_push_notifications: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_stream_sounds: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_desktop_notifications: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_sounds: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_offline_email_notifications: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_offline_push_notifications: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_online_push_notifications: Optional[bool]=REQ(validator=check_bool, default=None),
+        enable_digest_emails: Optional[bool]=REQ(validator=check_bool, default=None),
+        pm_content_in_desktop_notifications: Optional[bool]=REQ(validator=check_bool,
+                                                                default=None)) -> HttpResponse:
```

Comment:
You could use the same style that you used for `json_change_ui_settings` in line 62-62.